CHANGELOG.md

```diff
@@ -1,4 +1,4 @@
-# Unreleased
+# 2.0.4 (2021-12-07)
 
 * Added the ability to add additional settings to the configmap of the stardog configmap
```

Comment:
super small thing: this one is missing its corresponding issue

Reply:
Fixed. Thanks for spotting that.